bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -122,70 +128,128 @@ async def autoreview_loop(self) -> None:
-    async def list_command(
+    async def list_group(
         self,
         ctx: Context,
-        oldest_first: bool = False,
     ) -> None:
         """
-        Shows the users that are currently in the talent pool.
+        Shows the users that are currently in the talent pool, ordered by next to be reviewed.
 
-        The optional kwarg `oldest_first` can be used to order the list by oldest nomination.
+        Note that this order will change over time, so should not be relied upon.
         """
-        await self.list_nominated_users(ctx, oldest_first=oldest_first)
+        await self.list_nominated_users(ctx, order_by_priority=True)
+
+    @list_group.command(name="oldest")
+    async def list_oldest(self, ctx: Context) -> None:
+        """Shows the users that are currently in the talent pool, ordered by oldest nomination."""
+        await self.list_nominated_users(ctx, oldest_first=True)
+
+    @list_group.command(name='newest')
+    async def list_newest(self, ctx: Context) -> None:
+        """Shows the users that are currently in the talent pool, ordered by newest nomination."""
+        await self.list_nominated_users(ctx, oldest_first=False)
 
     async def list_nominated_users(
         self,
         ctx: Context,
         oldest_first: bool = False,
+        order_by_priority: bool = False
     ) -> None:
         """
-        Gives an overview of the nominated users list.
-
-        It specifies the user's mention, name, how long ago they were nominated, and whether their
-        review was posted.
+        Lists the currently nominated users.
 
         The optional kwarg `oldest_first` orders the list by oldest entry.
         """
+        now = datetime.now(tz=timezone.utc)
+        embed = Embed(
+            title="Talent Pool active nominations",
+            color=Color.blue()
+        )
         nominations = await self.api.get_nominations(active=True)
-        if oldest_first:
-            nominations = reversed(nominations)
 
-        lines = []
+        if not nominations:
+            embed.description = "There are no active nominations."
+            await ctx.send(embed=embed)
+            return
+
+        if order_by_priority:
+            nominations = await self.reviewer.sort_nominations_to_review(nominations, now)
+        elif oldest_first:
+            nominations.reverse()
+
+        messages_per_user = await self.api.get_activity(
+            [nomination.user_id for nomination in nominations],
+            days=RECENT_ACTIVITY_DAYS
+        )
 
+        lines: list[str] = []
         for nomination in nominations:
-            member = await get_or_fetch_member(ctx.guild, nomination.user_id)
             line = f"• `{nomination.user_id}`"
+
+            member = await get_or_fetch_member(ctx.guild, nomination.user_id)
             if member:
                 line += f" ({member.name}#{member.discriminator})"
+            else:
+                line += " (not in server)"
```

Comment:
I liked the way we crossed off the name before, it's more catchy that way.
Can't we combine them both ? e.g cross off the name **and** say that they're not part of our server anymore ?

Reply:
Given the 💤 idea below, you could use :userleave: from the bot's emojis in this case. Just a thought.

Reply:
I've left it as is for now so it's sort of symetrical with the `({member.name}#{member.discriminator})` that other's have, although i'm happy to change it.